scio-test/parquet/src/main/scala/com/spotify/scio/testing/parquet/parquet.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.testing
+
+import com.spotify.scio.testing.parquet.ParquetTestUtils._
+import magnolify.parquet.ParquetType
+import org.apache.avro.generic.GenericRecord
+import org.tensorflow.proto.example.Example
```

Comment:
I think this will crash if `tensorflow` is not part of the dependencies.
We need to split the packages to avoid runtime issues.

Reply:
I split the package objects, but I guess there would be the same issue with loading `ParquetTestUtils.scala` too, so I'll split that as well.